config/mysqlpersister/mysqlpersister.go

```diff
@@ -0,0 +1,210 @@
+package mysqlpersister
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+	"sync"
+	"time"
+
+	sq "github.com/Masterminds/squirrel"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/golang/protobuf/proto"
+	"github.com/jmoiron/sqlx"
+
+	"github.com/square/quotaservice/config"
+	"github.com/square/quotaservice/logging"
+	qsc "github.com/square/quotaservice/protos/config"
+)
+
+type mysqlPersister struct {
+	latestVersion int
+	db            *sqlx.DB
+	m             *sync.RWMutex
+
+	watcher  chan struct{}
+	shutdown chan struct{}
+
+	activeFetchers *sync.WaitGroup
+
+	configs map[int]*qsc.ServiceConfig
+}
+
+type configRow struct {
+	Version int    `db:"Version"`
+	Config  string `db:"Config"`
+}
+
+func New(dbUser, dbPass, dbHost string, dbPort int, dbName string, pollingInterval time.Duration) (config.ConfigPersister, error) {
+	db, err := sqlx.Open("mysql",
```

Comment:
Will this work if we need to use TLS to connect to the db?

Reply:
No, TLS requires extra steps.  Would we want to have that here or use ghosttunnel?
I could make it configurable.

Reply:
Configurable sounds great 😉